App/Redux/PreferencesRedux.ts

```diff
@@ -45,7 +45,8 @@ export const initialState: PreferencesState = {
   onboarded: false,
   verboseUi: false,
   tourScreens: {
-    wallet: true
+    wallet: true,
+    threads: true
```

Comment:
I don't understand why these would initially be `true`. Seems that the tour UI would never show for new users then. Maybe I'm missing something.

Reply:
true == yes, show it